backend/poc-backend/src/workflows/dto/workflow.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { Node, Edge } from './workflow-components.dto';
+/*Questa è la struttura dati salvata nel backend.
+  È possibile che sia presente un dto in formato non consistente es. solo archi o solo nodi.
+*/
+export class WorkflowDto {
```

Comment:
Non dovrebbe esserci un attributo che indica l'id del nodo da cui far partire l'esecuzione del workflow?

Reply:
dipende come lo vuoi gestire, se in un workflow mettiamo solo un'esecuzione lineare allora l'entrypoint sarà sempre il primo blocco

Reply:
Ok allora al frontend ci si assicura che il blocco con id = 1 sarà quello di start